internal/resources/metal/vlan/resource_test.go

```diff
@@ -63,6 +77,89 @@ func TestAccMetalVlan_metro(t *testing.T) {
 						"equinix_metal_vlan.foovlan", "metro", metro),
 				),
 			},
+			{
+				Config:   testAccCheckMetalVlanConfig_metro(rs, strings.ToUpper(metro), "tfacc-vlan"),
+				PlanOnly: true,
```

Comment:
This test step confirms that there are no changes to apply after changing the metro casing

Reply:
Any reason you don't just use upperMetro here?

Reply:
`metro` and `upperMetro` are being used in different ways, one is to use a different case as a starting point before converting case and the other is to have different metros to ensure that changing the metro field has the intended effect.